### PR TITLE
Migration Guide updates

### LINKED ISSUE #3973: No matching distribution found for tensorflow==1.12.0
@erohmensing, 

Rasa Core version: pip install rasa_core

Python version: 2.7

Operating system : windows 10

Issue:
facing error like,
 **ERROR: Could not find a version that satisfies the requirement tensorflow~=1.13.0 (from rasa_core) (from versions: none)
ERROR: No matching distribution found for tensorflow~=1.13.0 (from rasa_core)**

### PR DESCRIPTION
**Pull request by rgstephens**
**Proposed changes**:
Misc updates including #3793
- fixes #3973

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
Thanks a lot for submitting this PR 🙏 we'll give it a review as soon as possible

File: docs/migration-guide.rst [hunk 1]
```diff
@@ -32,6 +37,10 @@ General
 - If you were previously importing the ``Button`` or ``Element`` classes from
   ``rasa_core.dispatcher``, these are now to be imported from ``rasa_sdk.utils``.
 
+- The NLU and Core used `separate configuration files
```
Comment (erohmensing) [inline docs/migration-guide.rst]:
```suggestion
- Rasa NLU and Core previously used `separate configuration files 
```

File: docs/migration-guide.rst [hunk 2]
```diff
@@ -32,6 +37,10 @@ General
 - If you were previously importing the ``Button`` or ``Element`` classes from
   ``rasa_core.dispatcher``, these are now to be imported from ``rasa_sdk.utils``.
 
+- The NLU and Core used `separate configuration files 
+  <https://legacy-docs.rasa.com/docs/nlu/0.15.1/migrations/?&_ga=2.218966814.608734414.1560704810-314462423.1543594887#id1>`.
+  These two files should be merged into a single file that is passed via the `--config` parameter.
```
Comment (erohmensing) [inline docs/migration-guide.rst]:
```suggestion
  These two files should be merged into a single file either named ``config.yml``, or passed via the ``--config`` parameter.
```

**Comment by erohmensing** [comment]
@rgstephens would you update this? Would love to get this info into the guide and get your first contribution in!

**Comment by rgstephens** [comment]
Done.  Not my [first PR](https://github.com/RasaHQ/rasa/pull/1630) ;)

**Pull request merged**